[2024-02-27T23:12:22.363-0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_customer_dag.saving_to_csv manual__2024-02-28T04:09:53.320788+00:00 [queued]>
[2024-02-27T23:12:22.367-0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_customer_dag.saving_to_csv manual__2024-02-28T04:09:53.320788+00:00 [queued]>
[2024-02-27T23:12:22.368-0500] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2024-02-27T23:12:22.374-0500] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): saving_to_csv> on 2024-02-28 04:09:53.320788+00:00
[2024-02-27T23:12:22.378-0500] {standard_task_runner.py:57} INFO - Started process 49089 to run task
[2024-02-27T23:12:22.381-0500] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_customer_dag', 'saving_to_csv', 'manual__2024-02-28T04:09:53.320788+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/etl_customer_dag.py', '--cfg-path', '/var/folders/0j/ct19m_4d33qfzqs3tkf7s7rm0000gq/T/tmp3940nftj']
[2024-02-27T23:12:22.383-0500] {standard_task_runner.py:85} INFO - Job 85: Subtask saving_to_csv
[2024-02-27T23:12:22.464-0500] {task_command.py:416} INFO - Running <TaskInstance: etl_customer_dag.saving_to_csv manual__2024-02-28T04:09:53.320788+00:00 [running]> on host venkatasaismbp.mynetworksettings.com
[2024-02-27T23:12:22.564-0500] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Venkata' AIRFLOW_CTX_DAG_ID='etl_customer_dag' AIRFLOW_CTX_TASK_ID='saving_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2024-02-28T04:09:53.320788+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-28T04:09:53.320788+00:00'
[2024-02-27T23:12:22.592-0500] {python.py:194} INFO - Done. Returned value was: None
[2024-02-27T23:12:22.614-0500] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_customer_dag, task_id=saving_to_csv, execution_date=20240228T040953, start_date=20240228T041222, end_date=20240228T041222
[2024-02-27T23:12:22.640-0500] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-02-27T23:12:22.673-0500] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
